marlowe-dashboard-client/src/Marlowe/README.md

```diff
@@ -0,0 +1,38 @@
+# About Marlowe Contracts and the PAB
+
+"Contract" in the context of this app can mean several things:
+
+1. A Marlowe contract (a program that conforms to the schema defined in Marlowe.Semantics).
+2. An instance of a Marlowe contract installed on the blockchain. These can be identified
+   by their `MarloweParams`.
+3. A Plutus contract. Marlowe Run uses three Plutus contracts to operated (see below).
+4. An instance of a Plutus contract running in the PAB. These are identified by a
+   `ContractInstanceId`.
+
+## The Marlowe Plutus contracts
+
+Marlowe Run needs three Plutus `ContractType`s to be available in the PAB:
+
+1. `MarloweContract`: Every wallet has one instance of this contract installed in the PAB.
```

Comment:
This is the actual `Escrow`/`Swap`/etc contract, right? If that is so, why "Every wallet **has one instance** of this contract"

Reply:
No, this is the Marlowe *Plutus* contract, which is essentially just a program for managing Marlowe contracts.

Reply:
maybe we could rename to `MarlowePlutusContract`? And there is one per wallet or one per marlowe contract?